tests/addAndRemove.test.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @jest-environment jsdom
+ */
+import { addTodo, removeTodo } from '../src/addAndRemove.js';
+
+describe('adding a new todo list', () => {
+  document.body.innerHTML = '<input type="text" placeholder="Add to your list..." id="inputList">';
+  const inputList = document.getElementById('inputList');
+  inputList.value = 'Kossi';
+  const todos = [];
+
+  test('Add two new tasks to the list', () => {
+    addTodo(todos);
+    addTodo(todos);
+    expect(todos).toHaveLength(1);
+  });
+
+  test('Check description of first task', () => {
+    expect(todos[0].description).toBe('Kossi');
+  });
+
+  test('Check status of first task', () => {
+    expect(todos[0].completed).toBe(false);
+  });
+
+  test('Check index of second task', () => {
+    expect(todos[0].index).toBe(1);
+  });
+});
+
+describe('deleting a task from the list', () => {
+  const todos = [{
+    description: 'Walking the wire',
+    completed: false,
+    index: 1,
+  },
+
+  {
+    description: 'Go to supermarket',
+    completed: false,
+    index: 2,
+  },
+
+  {
+    description: 'Visit my family',
+    completed: false,
+    index: 3,
+  },
+
+  ];
+
+  test('Delete task with index 2', () => {
+    removeTodo(todos, 2);
+    expect((todos)).toHaveLength(3);
+  });
+
+  test('Update index after task deletion', () => {
+    expect(todos[2].index).toBe(3);
+  });
+});
```

Comment:
- [ ] Great Implementation, all tests are passing successfully, however, let us add few tests to for both Add and remove function
![arr5](https://user-images.githubusercontent.com/55635977/131876469-c7619a99-051a-47c5-98ad-255ae8f4e5fe.png)

`For Adding function`

Adds a task to a nonempty list; check description and index
Adds a task to an empty list; check description

`For Deleting func`

Update localStorage after deleting  elements
Update dom after deleting compeleted elements

Reply:
Hi Emmyn ,
Thank you for your comment but.
But there are some parts we don't understand cause I think we are implemented all the requirements and everything works fine.
 
![demo2](https://user-images.githubusercontent.com/6615394/131888064-6209a913-c655-4887-9989-621fc3bc0c73.PNG)
![demo-1](https://user-images.githubusercontent.com/6615394/131888075-0bb1bac6-611f-4a49-bbdd-c67abf09b0ef.PNG)

Can you please explain to us what were are supposed to do again, cause we don't understand your comments?
Cheers,

Reply:
We updated the project with jet.mock . You can check with the screenshot below. 

![moock](https://user-images.githubusercontent.com/6615394/131937395-2db603ab-d48e-43a5-9af0-ff63de24e11f.PNG)